tasks.py

```diff
@@ -684,7 +684,7 @@ def e2etest(ctx, name="kind", export=None, kubeconfig=None, system_namespaces="k
     ginkgo_focus = ""
     if focus:
         ginkgo_focus = "--ginkgo.focus=\"" + focus + "\""
-    
```

Comment:
Ah, is the removal of trailing white spaces okay in the same commit or should I remove this change?